static/js/index.js

```diff
@@ -1,14 +1,20 @@
-import {
-  addEvents,
-} from './hamburger.js';
-
+import addEvents from './hamburger.js';
 import showPopUp from './popUp.js';
+import {
+  removeErrorMessage,
+  validateEmail,
+} from './validators.js';
 
-window.addEventListener('resize', () => { window.location.reload(); });
-if (window.innerWidth < 992) {
-  addEvents();
+function main() {
+  window.addEventListener('resize', () => { window.location.reload(); });
+  if (window.innerWidth < 992) {
+    addEvents();
+  }
+  document.querySelectorAll('.see_more').forEach((button) => {
+    button.addEventListener('click', () => showPopUp(button.dataset.id));
+  });
+  document.querySelector('#contact-form').addEventListener('submit', validateEmail);
+  document.querySelector('input[name="email"]').addEventListener('input', removeErrorMessage);
 }
 
-document.querySelectorAll('.see_more').forEach((button) => {
-  button.addEventListener('click', () => showPopUp(button.dataset.id));
-});
\ No newline at end of file
+window.onload = main();
```

Comment:
[OPTIONAL] I think it should be just `window.onload = main;`. If you place the parenthesis after main, JS will execute main immediately after reading it instead of executing it in the `onload` event.

Reply:
Thank you